src/glmfit.jl

```diff
@@ -1,14 +1,28 @@
-type GlmResp{V<:FPVector,D<:UnivariateDistribution,L<:Link} <: ModResp       # response in a glm model
-    y::V                                       # response
+"""
+    GlmResp
+
+The response vector and various derived vectors in a generalized linear model.
+"""
+immutable GlmResp{V<:FPVector,D<:UnivariateDistribution,L<:Link} <: ModResp
+    "`y`: response vector"
+    y::V
     d::D
-    devresid::V                                # (squared) deviance residuals
-    eta::V                                     # linear predictor
-    mu::V                                      # mean response
-    mueta::V                                   # derivative of mu w.r.t. eta
-    offset::V                                  # offset added to linear predictor (usually 0)
-    wts::V                                     # prior weights
-    wrkwt::V                                   # working weights
-    wrkresid::V                                # working residuals
+    "`devresid`: the squared deviance residuals"
```

Comment:
I could be wrong, but I don't think you need to repeat the name of the field in the docstring.